Dockerfile

```diff
@@ -1,26 +1,24 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3-298.1618432845
+FROM registry.access.redhat.com/ubi9:9.2-755
 
 LABEL name="spaship/httpd" \
   summary="Apache Http Server for SPAship" \
   description="This a runtime image for SPAship apps" \
-  maintainer="Kun Yan <kyan@redhat.com>"
+  maintainer="Arkaprovo Bhattacharjee <arbhatta@redhat.com>"
 
 EXPOSE 8080
 
-RUN microdnf install -y --nodocs httpd && microdnf clean all
-
-RUN sed -i 's/Listen 80/Listen 8080/' /etc/httpd/conf/httpd.conf \
-  && sed -i 's/AllowOverride None/AllowOverride All/' /etc/httpd/conf/httpd.conf \
-  && chgrp -R 0 /var/log/httpd /var/run/httpd \
-  && chmod -R g=u /var/log/httpd /var/run/httpd
+RUN dnf install -y --nodocs --allowerasing coreutils httpd vim-minimal && \
```

Comment:
I don't know why it have to install `coreutils` and `vim-minimal`. 
I think these packages will help DevOps people to do debug work, but I think this kind of work are one time job, because all application within apache httpd should be static file, there will not have any application log

Keep the image works with simple and small is the better choice, right ?